content/en/community/maintainer_guide/supply-chain-security.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Add Supply Chain Security
```

Comment:
may be change this to `Add sbom generation step to release pipelines`

Reply:
Done